app/models/exceptions/resource_forbidden.rb

```diff
@@ -0,0 +1,5 @@
+class ResourceForbidden < ActionController::ActionControllerError
+  def status
+    403
```

Comment:
Można też użyć sybboli zamiast numerów: http://billpatrianakos.me/blog/2013/10/13/list-of-rails-status-code-symbols/

Tutaj ``:forbidden`` - kod będzie czytelniejszy